networkit/vizbridges.py

```diff
@@ -150,26 +194,55 @@ def widgetFromGraph(G, dimension = Dimension.Two, nodeScores = None, nodePartiti
 			raise Exception("Provide either nodeScores or nodePartition - not both.")
 		if len(nodeScores) != G.upperNodeIdBound():
 			raise Exception("nodeScores should include scores for every node.")	
+	
+	if edgeScores:
+		if isinstance(edgeScores, dict):
+			if not G.hasEdgeIds():
+				raise Exception("Edges need to be indexed to draw edge scores.")
+			edgeScoresList = [None] * G.numberOfEdges()
+			for u,v in G.iterEdges():
+				edgeScoresList[G.edgeId(u,v)] = edgeScores[(u,v)]
```

Comment:
This implicitly assumes that the index tuples of `edgeScores` (u,v) are stored in the same order that `G.iterEdges()` produces - if the index is (v,u) this code will not find the score.